salt/pillar/journal-cms.sls

```diff
@@ -16,6 +16,10 @@ journal_cms:
         queue: null
         endpoint: http://10.0.2.2:4100
         topic_template: arn:aws:sns:us-east-1:512686554592:bus-%s--dev
+
+    iiif:
+        base_uri: null
```

Comment:
There is a fallback in the code on journal-cms. I will reference it once that the PR is available.